docs/pages/example/mapbox-gl-geocoder-reverse-geocoder.js

```diff
@@ -0,0 +1,11 @@
+/*---
+title: Search using Coordinates or an Address
```

Comment:
We use sentence case for example titles.

```suggestion
title: Search using coordinates or an address
```